client/app/bundles/course/group/components/Note.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
```

Comment:
Do you think we can just put this component in the general component folder so that its easier to reuse in the future

Reply:
I think can, but perhaps in a subsequent PR